terraform/environments/analytical-platform-compute/kms-keys.tf

```diff
@@ -275,13 +275,33 @@ module "mlflow_s3_kms" {
   tags = local.tags
 }
 
-module "mojap_derived_tables_replication_s3_kms" {
+# module "mojap_derived_tables_replication_s3_kms" {
+#   #checkov:skip=CKV_TF_1:Module registry does not support commit hashes for versions
+#   #checkov:skip=CKV_TF_2:Module registry does not support tags for versions
+
+#   source  = "terraform-aws-modules/kms/aws"
+#   version = "3.1.1"
+
+#   aliases               = ["s3/mojap-derived-tables-replication"]
+#   description           = "mojap-derived-tables-replication S3 KMS key"
+#   enable_default_policy = true
+
+#   deletion_window_in_days = 7
+
+#   tags = local.tags
+# }
+
+module "mojap_derived_tables_replication_eu_west_1_s3_kms" {
   #checkov:skip=CKV_TF_1:Module registry does not support commit hashes for versions
   #checkov:skip=CKV_TF_2:Module registry does not support tags for versions
 
   source  = "terraform-aws-modules/kms/aws"
   version = "3.1.1"
 
+  providers = {
+    aws = aws.analytical-platform-compute-eu-west-1
+  }
+
   aliases               = ["s3/mojap-derived-tables-replication"]
```

Comment:
Can you please add the region to the alias? It's good practice anyway but KMS has a 7-day key deletion reversion window so you might actually hit an issue trying to reuse the alias.